src/Forside/SammenligningMedBransje/EkspanderbartSammenligningspanel.tsx

```diff
@@ -63,32 +66,30 @@ export const EkspanderbartSammenligningspanel: FunctionComponent<Props> = (
             !erIaTjenesterMetrikkerSendtForBedrift(orgnr, context.bedrifterSomHarSendtMetrikker) &&
             erÅpen
         ) {
-            sendIaTjenesteMetrikkEvent()
-                .then((isSent) => {
-                    if (isSent) {
-                        context.setBedrifterSomHarSendtMetrikker(
-                            iaTjenesterMetrikkerErSendtForBedrift(
-                                orgnr,
-                                context.bedrifterSomHarSendtMetrikker,
-                            ),
-                        );
-                    }
-                });
+            sendIaTjenesteMetrikkEvent().then((isSent) => {
+                if (isSent) {
+                    context.setBedrifterSomHarSendtMetrikker(
+                        iaTjenesterMetrikkerErSendtForBedrift(
+                            orgnr,
+                            context.bedrifterSomHarSendtMetrikker
+                        )
+                    );
+                }
+            });
         }
-    }, [erÅpen]);
+    }, [erÅpen, context, orgnr, sendIaTjenesteMetrikkEvent]);
```

Comment:
Hvorfor legge til disse ? :)

Reply:
Fordi variabel/funksjonene blir brukt i useEffecten uten at de er lagt inn i dependency-arrayen. Ref https://typeofnan.dev/you-probably-shouldnt-ignore-react-hooks-exhaustive-deps-warnings/

Reply:
Hvis man ikke gjør det så får man en advarsel fra eslint `react-hooks/exhaustive-deps`